src/systems/hubs-systems.js

```diff
@@ -41,10 +44,25 @@ AFRAME.registerSystem("hubs-systems", {
     this.hapticFeedbackSystem.tick(this.twoPointStretchingSystem, this.singleActionButtonSystem.didInteractThisFrame);
     this.soundEffectsSystem.tick();
     this.lobbyCameraSystem.tick();
-    // batchManager is ticked in "post-physics" system
   },
 
   remove() {
     this.cursorTargettingSystem.remove();
   }
 });
+
+const i = window.setInterval(() => {
+  if (AFRAME.scenes[0] && AFRAME.scenes[0].systems && AFRAME.scenes[0].systems.physics) {
```

Comment:
this seems pretty hacky -- can't we always register and move this check lazily into `tick`?

Reply:
ah i see this is how it was before -- still hacky :)

Reply:
This will go away will Kevin's upcoming changes to the physics system. The issue we have before then is that we don't want to register this system until after the physics system registers itself (because registration order == tick order), and the physics system registration happens asynchronously (hence this hack). Planning to leave it as-is since it'll get removed soon.